webapp/src/main/webapp/news-list-view/components/views/NewsMosaicView.vue

```diff
@@ -76,20 +87,24 @@ export default {
         month: 'long',
         day: 'numeric',
       },
+      isSmallWidth: false
     };
   },
   created() {
     this.reset();
     this.$root.$on('saved-news-settings', this.refreshNewsViews);
     this.getNewsList();
   },
+  updated() {
+    this.isSmallWidth =  this.$refs.top?.clientWidth *100 / window.screen.width  < 33;
```

Comment:
Why are you using this check ? couldn't we use Mobile vs Tablet vs Dersktop?

Reply:
this method allows to detect the dimension of the screen and to check if the display of the mosaic template is by column or by the normal form.